docs/index.txt

```diff
@@ -38,17 +38,8 @@ Having trouble? We'd like to help!
 * Looking for specific information? Try the :ref:`genindex`, :ref:`modindex` or
   the :doc:`detailed table of contents <contents>`.
 
-* Search for information in the archives of the |django-users| mailing list, or
-  `post a question`_.
-
-* Ask a question in the `#django IRC channel`_.
-
-* Report bugs with Django in our `ticket tracker`_.
-
-.. _archives: https://groups.google.com/group/django-users/
-.. _post a question: https://groups.google.com/d/forum/django-users
-.. _#django IRC channel: irc://irc.freenode.net/django
-.. _ticket tracker: https://code.djangoproject.com/
+* Not found anything? See :doc:`faq/help` for information on asking questions
+  to the community.
```

Comment:
I'd suggest we labour the point a little here...

```suggestion
* Not found anything? See :doc:`faq/help` for information on getting support 
  and asking questions to the community.

```

... because I think we need to keep the link to Trac on the homepage. 

We could reword that last to emphasise "You've found a bug?", but TBH we don't get a lot of support requests on the issue tracker. (Some, yes, but...)